Update pmw_89test to use xarray_to_xarray format

mindyls opened the pull request:
NOTE: apply_data_range does not currently work for DataArrays, need to fix that.  For now converting to numpy arrays internal to this algorithm.

Package: my_package
Total run time: 176 seconds
Number data types run: 5
Number data types failed: 0

Package: geoips_full
Total run time: 2588 seconds
Number data types run: 47
Number data types failed: 0

jsolbrig commented:
@mindyls I added a temporary fix for `apply_data_range` in geoips#307 and used it in #16.

jsolbrig reviewed (dismissed):
This looks good to me. I added a fix for `apply_data_range` in geoips#307 and #16. I also added issue #308 to remove that band-aid and do it correctly.

mindyls commented:
> This looks good to me. I added a fix for `apply_data_range` in geoips#307 and #16. I also added issue #308 to remove that band-aid and do it correctly.

I'm not sure we want to make that change at this point for the workshop - I'll let @christopher-selman decide :-)
